### PR TITLE
pass kwargs to minimizer

### PR DESCRIPTION
**Pull request by juanitorduz**
Closes https://github.com/pymc-labs/pymc-marketing/issues/729

----
📚 Documentation preview 📚: https://pymc-marketing--737.org.readthedocs.build/en/737/

### CODE REVIEW

File: tests/mmm/test_budget_optimizer.py [hunk 1]
```diff
@@ -34,14 +51,24 @@
                     "saturation_params": {"lam": 20, "beta": 1.0},
                 },
             },
+            {
+                "method": "SLSQP",
+                "options": {"ftol": 1e-8, "maxiter": 1_002},
+            },
```
Comment (wd60622) [inline tests/mmm/test_budget_optimizer.py:57]: Is there any check or sign that this is actually being used? Is there something we can check?

**Review by wd60622 — COMMENTED** [review]
Not the end of the world since the code looks fine

File: tests/mmm/test_budget_optimizer.py (hunk 1 above)
Comment (juanitorduz) [reply tests/mmm/test_budget_optimizer.py:60]: Good remark! Added a specific test in https://github.com/pymc-labs/pymc-marketing/pull/737/commits/7607668bb96899d1badabd2617ec7f26d2c58550

**Review by wd60622 — APPROVED** [review]
Looks good

**Pull request merged**